hc/api/tests/test_create_check.py

```diff
@@ -78,4 +91,27 @@ def test_it_rejects_non_string_name(self):
                   expected_error="name is not a string")
 
     ### Test for the assignment of channels
+    def test_assignment_of_channels(self):
+        check = Check()
```

Comment:
Include an assertion when there's existing channels. Also assign the check to a user